### ISSUE TITLE
Broken link in Python grader tests

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#11061]
In the Python grader code, the methods `check_numpy_array_sanity` and `check_numpy_array_features` have a check for `np.matrix` that leads to a shortened link:
https://github.com/PrairieLearn/PrairieLearn/blob/d72ad482062753f38e765d182d8dd4d9dff516d5/graders/python/python_autograder/code_feedback.py#L139-L143

The link `bit.ly/array-vs-matrix` is currently broken. It points to:
```
https://docs.scipy.org/doc/scipy/reference/tutorial/linalg.html#numpy-matrix-vs-2d-numpy-ndarray
```
The corresponding documentation seems to have been moved to:
```
https://docs.scipy.org/doc/scipy/tutorial/linalg.html#numpy-matrix-vs-2-d-numpy-ndarray
```

I don't know who owns the bit.ly link above, but I'd argue that it'd make more sense to either change this link to a full link or to describe a summary of the issue inline instead.

**Comment by nwalters512** [comment]
Agreed, let's change the link. Please open a PR, that'd probably take less time than writing up this issue 😛

**Comment by jonatanschroeder** [comment]
This is currently conflicting with #11059, I'll wait until that is merged to fix it (unless @reteps wants to add it there...).

**Comment by jonatanschroeder** [comment]
Resolved in #11050.
